ppr-api/src/ppr_api/resources/financing_statements.py

```diff
@@ -886,7 +886,17 @@ def pay_and_save(req: request,  # pylint: disable=too-many-arguments,too-many-lo
         pay_trans_type = TransactionTypes.DISCHARGE.value
 
     if not is_reg_staff_account(account_id):
-        pay_account_id: str = account_id if not is_sbc_office_account(account_id) else None
+        pay_account_id: str = account_id
+        # if gov user then check if sbc staff
+        if is_gov_account(jwt):
+            # if SBC staff user (authy, api call) then set payment account id to None
+            is_sbc = is_sbc_office_account(jwt.get_token_auth_header(), account_id)
+            if is_sbc:
+                pay_account_id = None
+            elif is_sbc == None:
```

Comment:
I just had a question about this... if the user isn't sbc does it raise an exception?

Reply:
if the call returns successfully and they are not sbc staff then it will return false. None happens if there was an issue or if there's a case where they didn't have an account id